.jules/runs/steward_release/pr_body.md

```
@@ -0,0 +1,61 @@
+## 💡 Summary
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)